app/src/main/java/org/oppia/app/recyclerview/OnItemDragListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.recyclerview
+
+import androidx.recyclerview.widget.RecyclerView
+
+/** Listener for handling action of  [DragItemTouchHelperCallback]. */
+interface OnItemDragListener {
+  fun onItemDragged(indexFrom: Int, indexTo: Int, adapter: RecyclerView.Adapter<RecyclerView.ViewHolder>)
```

Comment:
Please add documentation: when will this be called? What do the values mean? What are implementations of this method expected to do?

Reply:
done